third_party/move/move-compiler-v2/tests/checking/dotdot/extra_dotdot.move

```diff
@@ -0,0 +1,7 @@
+module 0x42::test {
+    struct S(bool, u8, address);
+
+    fun extra_dotdot(x: S) {
+        let S(x, _, _, ..) = x;
```

Comment:
Let's also please test:
1. extra dotdot, but at the beginning of `S`
2. extra dotdot, but at the middle (`let S(_, .., _, _) = x`)
2. extra dotdot for structs with named fields.

Reply:
done